packages/sources/tradingeconomics-test/src/endpoint/price.ts

```diff
@@ -0,0 +1,52 @@
+import { makeLogger } from '@chainlink/external-adapter-framework/util'
+import { HttpTransport } from '@chainlink/external-adapter-framework/transports'
+import { EndpointTypes } from './price-router'
+
+const logger = makeLogger('TradinEconomics HTTP')
+
+export const batchTransport = new HttpTransport<EndpointTypes>({
+  prepareRequests: (params, config) => {
+    return params.map((symbol) => {
```

Comment:
it seems that trading economics supports batch requests with multiple symbols. If so we need to refactor this part